src/MapPanel.tsx

```diff
@@ -301,8 +303,8 @@ export class MapPanel extends Component<Props> {
         ]).then((topologyData) => {
           for(let i=0; i<LAYER_LIMIT; i++){
             try {
-              let parsedData = parseData(data, topologyData[i], colors[i], fields[i], i);
-              topology[i] = parsedData[3];
+              // @ts-ignore
```

Comment:
ts-ignore here to let JS do its thing -- this is a weird string/object/nully set of things that can't be typed easily